package/libs/libxml2/Makefile

```diff
@@ -192,7 +192,7 @@ define Package/libxml2-dev/install
 	$(SED) "s,$(STAGING_DIR),,g" $(1)/usr/lib/pkgconfig/*.pc
 
 	$(INSTALL_DIR) $(1)/usr/share/aclocal
-	$(CP) $(PKG_INSTALL_DIR)/usr/share/aclocal/* $(1)/usr/share/aclocal
+	$(CP) $(PKG_BUILD_DIR)/aclocal.m4 $(1)/usr/share/aclocal
```

Comment:
Please do not install the `aclocal.m4` file. Maybe this is not needed at all any more.

Reply:
I think this was related to the removed feature: https://github.com/GNOME/libxml2/commit/e714f506e488d0d20bd28a7d8de1ff2ecf43c9e8